bridges/ethElrond/steps/elrondToEth/step01GetPending.go

```diff
@@ -33,6 +33,17 @@ func (step *getPendingStep) Execute(ctx context.Context) core.StepIdentifier {
 		return step.Identifier()
 	}
 
+	isValid, err := step.bridge.ValidateBatch(batch.Json())
+	if err != nil {
+		step.bridge.PrintInfo(logger.LogError, "error validating Elrond batch", "error", err)
```

Comment:
maybe print also the json formatted batch as to easily debug? Valid also on L43 + ethToElrond/step01GetPending.go

Reply:
added